src/boot/store.js

```diff
@@ -163,6 +163,15 @@ const migrations: { [string]: (GlobalState) => GlobalState } = {
         typeof a.zulipVersion === 'string' ? new ZulipVersion(a.zulipVersion) : a.zulipVersion,
     })),
   }),
+
+  // Accounts.zulipFeatureLevel is `number | null`
```

Comment:
One thing that would help in these comments is that it's just as helpful (or almost as helpful) to know the *old* status of the bit of data in question as the new status.

So e.g. for this one: "Added Accounts.zulipFeatureLevel as `number | null`"